tests/src/test_cases/widgets/test_bar.c

```diff
@@ -321,50 +321,50 @@ void test_bar_indicator_should_be_drawn_towards_the_min_range_side_after_setting
 static lv_obj_t * styled_bar_create(bool ver, int32_t start_value, int32_t end_value, lv_grad_dir_t grad_dir,
                                     int32_t bg_radius, int32_t indic_radius, int32_t bg_pad)
 {
-    lv_obj_t * bar = lv_bar_create(lv_screen_active());
-    if(ver) lv_obj_set_size(bar, 20, 100);
-    else lv_obj_set_size(bar, 100, 20);
-    lv_bar_set_range(bar, 0, 100);
-    lv_bar_set_mode(bar, LV_BAR_MODE_RANGE);
-    lv_bar_set_value(bar, end_value, LV_ANIM_OFF);
-    lv_bar_set_start_value(bar, start_value, LV_ANIM_OFF);
-
-    lv_obj_set_style_bg_opa(bar, 255, LV_PART_MAIN);
-    lv_obj_set_style_bg_color(bar, lv_color_hex3(0x0ff), LV_PART_MAIN);
-    lv_obj_set_style_radius(bar, bg_radius, LV_PART_MAIN);
-    lv_obj_set_style_pad_all(bar, bg_pad, LV_PART_MAIN);
-
-    lv_obj_set_style_bg_opa(bar, LV_OPA_70, LV_PART_INDICATOR);
-    lv_obj_set_style_bg_color(bar, lv_color_hex3(0xf0f), LV_PART_INDICATOR);
-    lv_obj_set_style_bg_grad_color(bar, lv_color_hex3(0x8f8), LV_PART_INDICATOR);
-    lv_obj_set_style_bg_grad_dir(bar, grad_dir, LV_PART_INDICATOR);
-
-    lv_obj_set_style_border_width(bar, 2, LV_PART_INDICATOR);
-    lv_obj_set_style_border_color(bar, lv_color_hex3(0x0f0), LV_PART_INDICATOR);
-    lv_obj_set_style_border_opa(bar, LV_OPA_70, LV_PART_INDICATOR);
-    lv_obj_set_style_outline_width(bar, 2, LV_PART_INDICATOR);
-    lv_obj_set_style_outline_pad(bar, 4, LV_PART_INDICATOR);
-    lv_obj_set_style_outline_color(bar, lv_color_hex3(0xff0), LV_PART_INDICATOR);
-    lv_obj_set_style_outline_opa(bar, LV_OPA_70, LV_PART_INDICATOR);
-    lv_obj_set_style_shadow_width(bar, 20, LV_PART_INDICATOR);
-    lv_obj_set_style_shadow_spread(bar, 5, LV_PART_INDICATOR);
-    lv_obj_set_style_shadow_color(bar, lv_color_hex3(0xf00), LV_PART_INDICATOR);
-    lv_obj_set_style_shadow_opa(bar, LV_OPA_70, LV_PART_INDICATOR);
-    lv_obj_set_style_radius(bar, indic_radius, LV_PART_INDICATOR);
-
-    return bar;
+    lv_obj_t * bar_ = lv_bar_create(lv_screen_active());
```

Comment:
Why change this? Is it a warning because of a name collision between a local variable and a global variable?